src/compiler/prioritization/priority-levels.md

```diff
@@ -0,0 +1,96 @@
+# Priority levels
+
+As the compiler team's resources are limited, the prioritization working group's main goal is to identify the most relevant issues to work on, so that the compiler team can focus on what matters the most.
+
+## Words used in this document:
+
+`issue` refers to bugs and feature requests that are nominated for prioritization, by flagging the `I-Prioritize` label as described below.
+
+This document will define what each label means, and what strategy for each label will be used.
+
+## Labels
+
+Labeling an issue as `I-Prioritize` starts the prioritization process, which will end by removing the `I-Prioritize` label and appending one of the 4 labels we will discuss below:
+
+- P-Critical
+- P-High
+- P-Medium
+- P-Low
+
+Each of these labels defines a strategy the team will adopt regarding:
+
+- The amount of focus a given issue will receive
+- How members of the community can get involved
+
+## Definitions
+
+### P-Critical
+
+A `P-Critical` issue is a potentially blocker issue.
+
+The Working Group will keep track of the number of such issues open at the same time (which shouldn't be more than 5), and how long such issues remain open.
+
+Examples of things we typically judge to be “critical” bugs:
+
+- Regressions where code that used to compile no longer does
+
+  - Mitigating conditions that may lower priority:
+    - If the code should never have compiled in the first place (but if the regression affects a large number of crates, this may indicate that we need a warning period)
+    - If the code in question is theoretical and considered unlikely to exist in the wild, or if it only exists in small, unmaintained packages that are not widely used
+  - If a regression has been in stable for a release or two (either because we are still awaiting a fix, or because the bug had laid dormant i.e. undetected), we typically lower the priority as well, because by that time, if the users have not raised a ruckus about the regression, that is a sign that it is inherently not a critical issue. Eg: [an issue that would have been P-Critical but ended up being P-High](https://rust-lang.zulipchat.com/#narrow/stream/227806-t-compiler.2Fwg-prioritization/topic/pre-meeting.20triage.202020-04-09.20.2354818)
+
+- Regressions where code still compiles but does something different than it used to do (dynamic semantics have changed)
+  - Mitigating conditions that may lower priority:
+    - If code uses feature that is explicitly not specified (e.g. `std::vec::Vec` docs state order in which it drops its elements is subject to change)
+- Feature-gated features accessible without a feature gate
+  - Mitigating conditions that may lower priority:
+    - If the pattern is VERY unlikely
+- Soundness holes where common code that should not compile actually does
```

Comment:
I think miscompilations are covered by the first bullet, "Regressions where code still compiles but does something different than it used to do (dynamic semantics have changed)", ...

... though arguably that is "solely" talking about regressions. It might be better to talk about deviations from the *intended semantics* (rather than the historical semantics)